src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -110,6 +118,19 @@ def load_arguments(self, _):
     with self.argument_context('resource tag') as c:
         c.argument('is_incremental', action='store_true', options_list=['--is-incremental', '-i'],
                    help='The option to add tags incrementally without deleting the original tags. If the key of new tag and original tag are duplicated, the original value will be overwritten.')
+        c.argument('latest_include_preview', latest_include_preview_type)
```

Comment:
what about adding in `resource` and ignoring in specific command?

Reply:
Thanks for your reminding~  I did it at first, but had some problems. Now add the following code to solve the problem, so update it~
```
with self.argument_context('resource wait') as c:
        c.ignore('latest_include_preview')
```